src/app/static/src/tests/filteredData/getters.test.ts

```diff
@@ -16,24 +16,8 @@ import {
 } from "../mocks";
 import {flattenOptions, getUnfilteredData} from "../../app/store/filteredData/utils";
 
-export function testGetters(state: FilteredDataState, rootState = mockRootState()) {
-    const self = {
-        colorFunctions: {
-            art: function (t: number) {
-                return `rgb(${t},0,0)`;
-            },
-            prev: function (t: number) {
-                return `rgb(0,${t},0)`;
-            }
-        },
-        choroplethRanges: {
-            prev: {min: 0, max: 1},
-            art: {min: 0, max: 1}
-        }
-    } as any;
-
-    self.excludeRow = getters.excludeRow(state, self, rootState);
-    return self;
+export function testGetters(state: FilteredDataState) {
+    return {excludeRow: getters.excludeRow(state)};
```

Comment:
The other functions being set up (colorFunctions and choroplethRanges) were out of date/obsolete.